Gopkg.lock

```diff
@@ -2,139 +2,104 @@
 
 
 [[projects]]
-  digest = "1:2aff5edb9bccd2974090fddb17ca7ab05a3f5c983db567c30c7f0b53404f5783"
```

Comment:
What dep version are you using? Looks like something <0.5?

Reply:
no, I'm using `0.5.0`:
```
$ dep version
dep:
 version     : v0.5.0
 build date  : 2018-07-26
 git hash    : 224a564
 go version  : go1.10.3
 go compiler : gc
 platform    : darwin/amd64
 features    : ImportDuringSolve=false
```